packages/algob/src/lib/files.ts

```diff
@@ -37,19 +37,43 @@ export function assertDirectDirChildren (dir: string, scriptNames: string[]): st
 /**
  * Description: this function reads raw signed txn from file /assets/<filename>.tx
  * and returns the encoded txn as Uint8array
- * @param filename : filename [must have .tx ext]
+ * @param fileName : fileName [must have .tx ext]
  * @returns signed transaction encoded as Uint8array
  */
-export function loadSignedTxnFromFile (filename: string): Uint8Array | undefined {
-  if (!filename.endsWith(txExt)) {
-    throw new Error(`filename "${filename}" must end with "${txExt}"`);
+export function loadSignedTxnFromFile (fileName: string): Uint8Array | undefined {
+  if (!fileName.endsWith(txExt)) {
+    throw new Error(`filename "${fileName}" must end with "${txExt}"`);
   }
   try {
-    const p = path.join(ASSETS_DIR, filename);
+    const p = path.join(ASSETS_DIR, fileName);
     const buffer = fs.readFileSync(p);
     return Uint8Array.from(buffer);
   } catch (e) {
     if (e?.errno === -2) return undefined; // handling a not existing file
     throw e;
   }
 }
+
+/**
+ * Description: This function reads raw signed transaction from file /assets/<filename.ext>
+ * and returns the encoded txn as Uint8Array Buffer
+ * @param fileName : fileName
+ * @returns signed transaction as buffer
+ */
+export function loadRawSignedTxnFromFile (fileName: string): Buffer | undefined {
+  const p = path.join(ASSETS_DIR, fileName);
+  const txFile = fs.readFileSync(p);
+  if (txFile === undefined) {
+    throw new Error(`File ${fileName} does not exist`);
+  }
+  return txFile;
+}
+
+/**
+ * Description: This function writes the data to file.
+ * @param fileName: Output file name
+ * @param data: Data to be written to file
+ */
+export function writeToFile (fileName: string, data: any): void {
+  fs.outputFileSync(fileName, data);
```

Comment:
a) This should be in try catch as well (as it could fail). We need to also check for write permission here.
b) I think we have a similar function in `builtin-tasks/genaccount`. Could you please check?

Reply:
okay

Reply:
I'm changing this to use the function from `builtin-task/genaccount`. Thanks for this.